### PR TITLE
Add little-cms

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/little-cms/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - jpeg
+    - zlib
+    - libtiff
+  run:
```
Comment (marcelotrevisani) [inline recipes/little-cms/meta.yaml]: Could you please delete the empty section?

File: recipes/little-cms/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  url: https://github.com/mm2/Little-CMS/archive/lcms{{version}}.tar.gz
+  sha256: 8e23a09dc81af856db37941a4ea26acdf6a45b0281ec5b7ee94b5a4e9f7afbf7
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (chrisburr) [inline recipes/little-cms/meta.yaml]:
```suggestion
    - make
```

**Comment by isuruf** [comment]
See https://github.com/conda-forge/lcms2-feedstock/blob/master/recipe/meta.yaml

**Comment by sebastian-luna-valero** [comment]
Great, many thanks!

**Pull request closed without merging**